### DISCUSSION TITLE
Choosing reference samples for batch correction

### DISCUSSION THREAD

**Discussion by tammyyyx** [ImmuneDynamics/Spectre#12] [Q&A] [answered]
Hi Thomas,

In the batch alignment tutorial from the general discovery workflow, Mock_01 from batch A and Mock_05 from batch B are chosen to be the reference samples for batch correction and alignment.  Are they from the same donor? If not, what is the rationale behind choosing different donors for  batch correction? And how should we select the ideal reference sample from batches?

Thanks!

Regards,

Tammy

**Comment by tomashhurst** [comment, marked as answer]
Hi Tammy,

That's an excellent question. In this case, they are from normal mice and so can be cautiously used as a form of reference control. Because the mice are genetically identical (and from the same animal house), they are very consistent with each other. 

It's a slightly different use case then the typical setup (i.e. multiple PBMC samples from a single donor), but it does the trick.

**Comment by tammyyyx** [comment]
That makes sense. Thanks Thomas!
